assets/src/api.ts

```diff
@@ -249,6 +253,21 @@ export const fetchDetourDirections = (
     })
   ).then((v) => map(v, detourShapeFromData))
 
+export const fetchUnfinishedDetour = (
+  routePatternId: RoutePatternId,
+  connectionStart: ShapePoint
+): Promise<UnfinishedDetour | null> =>
+  checkedApiCall<UnfinishedDetourData, UnfinishedDetour | null>({
```

Comment:
is there a reason to prefer `checkedApiCall` over maybe `apiCallResult`?

Reply:
We could probably mark those other ones as deprecated (in another PR), thoughts?

Reply:
Not _immediately_ opposed. Would be interested in thinking more about it later.

Reply:
My preference is to keep this and `fetchFinishedDetour` the same. Let's change them both over as a follow-up?